26-register_publication/1214-creator_registers_corresponding_author.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Creator registers Corresponding Author
+
+    @1214
+    Scenario: Creator registers Corresponding Author
+        Given Creator begins registering a Publication in the Wizard
+        And they navigate to the Contributors tab
+        And they see one Author
+        When they check the Corresponding checkbox
+        And they see the Corresponding Author Email field
+        And they enter a valid email address in the Corresponding Author Email field
+        And they navigate to the Submission tab
+        Then they see that the Author is registered as the Corresponding Author
```

Comment:
får litt følelsen av at en publikasjon bare kan ha en corresponding author når jeg leser "(...) as *the* Corresponding Author", men det kan vel like godt være mange?

Reply:
har kanskje sammenheng med "And they see **one** Author"...
syns `a Author` og `a Corresponding Author` hadde passet bedre

Reply:
Ja, skjønner. Kan godt endre til `an Author`, men skjønner ikke hvor vil du ha `a Corresponding Author`?

Reply:
mener du isteden for `the Corresponding Author`?

Reply:
Bruk gjerne suggestions 😁